### ISSUE TITLE
Officially supported open source logs exporter aside from OTLP

### ISSUE THREAD

**Issue by codeboten** [open-telemetry/opentelemetry-specification#2206]
**What are you trying to achieve?**

Metrics and tracing have information about officially supported exporters which must be implemented in every language in addition to OTLP. This issue is really a question of whether or not there have been discussions on additional backends supported via exporters for the logging signal.

**Comment by tigrannajaryan** [comment]
@codeboten any ideas on what we could support? Unlike traces where Jaeger and Zipkin appeared to be the most popular open source tracing systems I can't think of an equivalent protocol in the logging world. Sure, there is for example Syslog, but it is a very limited protocol, I wouldn't want to make it an "exemplary" protocol that we support.

**Comment by codeboten** [comment]
I wonder if either [OpenSearch](https://github.com/opensearch-project/OpenSearch) or [Loki](https://github.com/grafana/loki) would be suitable for this, I only have limited experience with those.

**Comment by tigrannajaryan** [comment]
> I wonder if either [OpenSearch](https://github.com/opensearch-project/OpenSearch) or [Loki](https://github.com/grafana/loki) would be suitable for this, I only have limited experience with those.

To be honest neither of these strike me as as widely used and as influental a Jarger and Zipkin are in the tracin world. I think logging worlds is so diverse and fragmented that it is likely not possible to identify a couple formats which likely cover the majority of OSS use cases like Jarger and Zipkin (I think) do. This is all of course just my subjective opinion, I don't know if an actual industry-wide usage data is possible to obtain to try to decide objectively.

Unless we can identify 1-2 very strong undeniable candidates I would prefer to avoid supporting any, so that that we can keep our neutrality stance.

**Comment by codeboten** [comment]
Fair enough, I'm not 100% sure whether there is a strong candidate here either (which was the original reason for creating the issue).

Having jaeger/zipkin for tracing and prometheus for metrics as supported backends has allowed us to validate that the data model and implementations across languages works end to end with existing open source software, which I think is quite beneficial to end users and the project.

**Comment by tigrannajaryan** [comment]
> Having jaeger/zipkin for tracing and prometheus for metrics as supported backends has allowed us to validate that the data model and implementations across languages works end to end with existing open source software, which I think is quite beneficial to end users and the project.

Fair. I think at this point we have sufficient receivers/exporters in the Collector to validate the log data model, so I am not too worried about it.

**Comment by pmm-sumo** [comment]
I think having fluent forward exporter might be helpful in a way similar to having Jaeger or Zipkin for tracing. We could reuse the [receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/fluentforwardreceiver) code or use some 3rd party library, e.g. https://github.com/IBM/fluent-forward-go

**Comment by tigrannajaryan** [comment]
> I think having fluent forward exporter might be helpful in a way similar to having Jaeger or Zipkin for tracing.

I would like to see some community support before we do this. I am not sure fluentforward is so popular that it warrants the effort. We implemented the fluentforward protocol in the Collector but I don't recall seeing a lot of people using it or reporting issues about it.

**Comment by tigrannajaryan** [comment]
Closing as we haven't seen further interest in this.
